### DISCUSSION TITLE
Battery Statistics based in kWh and degradation

### DISCUSSION THREAD

**Discussion by fishersolutions** [teslamate-org/teslamate#1547] [Ideas]
I would be nice to have an improvement on the range page that showed the battery data in kWh.  (capacity)
then a %degradation over time.

**Comment by adriankumpf** [comment]
Feature suggestions belong here: [💡 Ideas](https://github.com/adriankumpf/teslamate/discussions/categories/ideas). I'll move it

**Reply by MarcFinns** [reply]
A battery degradation chart would be really great indeed..

**Comment by regenerous** [comment]
I second this. Something similar to this would be great
![Ew-pYZdXMAAqXKK](https://user-images.githubusercontent.com/6217175/139565758-367c6317-037e-4cf6-a9be-b456b91f1923.jpg)

**Comment by donker** [comment]
Does anyone have a pointer on how to calculate this? I'd love to have the battery deg as a nr somewhere easily accessible. Like maybe on the homepage of the app.

**Comment by RicoSuter** [comment]
Any update on this? This would be great..
